commit 5f9a609f278a6b3934b46f036ec671ecfe15fadc
Author: Samuel Thibault <samuel.thibault@ens-lyon.org>
Date:   Tue Apr 7 11:01:08 2020 +0200

    brlapi server: Fix build with -fno-common and --disable-contracted-braille

diff --git a/Programs/brlapi_server.c b/Programs/brlapi_server.c
index 242b1f0ea..8136d4ce2 100644
--- a/Programs/brlapi_server.c
+++ b/Programs/brlapi_server.c
@@ -1964,6 +1964,7 @@ PARAM_WRITER(computerBrailleTable)
   return param_writeString(changeTextTable, data, size);
 }
 
+#ifdef ENABLE_CONTRACTED_BRAILLE
 /* BRLAPI_PARAM_LITERARY_BRAILLE_TABLE */
 PARAM_READER(literaryBrailleTable)
 {
@@ -1975,6 +1976,7 @@ PARAM_WRITER(literaryBrailleTable)
 {
   return param_writeString(changeContractionTable, data, size);
 }
+#endif
 
 /* BRLAPI_PARAM_MESSAGE_LOCALE */
 PARAM_READER(messageLocale)
@@ -2173,11 +2175,13 @@ static const ParamDispatch paramDispatch[BRLAPI_PARAM_COUNT] = {
     .write = param_computerBrailleTable_write,
   },
 
+#ifdef ENABLE_CONTRACTED_BRAILLE
   [BRLAPI_PARAM_LITERARY_BRAILLE_TABLE] = {
     .global = 1,
     .read = param_literaryBrailleTable_read,
     .write = param_literaryBrailleTable_write,
   },
+#endif /* ENABLE_CONTRACTED_BRAILLE */
 
   [BRLAPI_PARAM_MESSAGE_LOCALE] = {
     .global = 1,
